AppHandling/Run-AlPipeline.ps1

```diff
@@ -838,7 +939,11 @@ if ($bcptTestFolders) {
 
 $artifactUrl = ""
 $filesOnly = $false
-if ($bcAuthContext) {
+$IsBcSaaSInfrastructure = $bcAuthContext -and $bcAuthContext -is [Hashtable] -and $bcAuthContext.ContainsKey('scopes') -and $bcAuthContext.scopes -like "https://projectmadeira.com/*"
+if ($IsBcSaaSInfrastructure) {
+    Write-Host "Using BC SaaS Infrastructure. Test for feature compatibility."
+}

```

Comment:
Write-Host could be moved to the if statement jut below

Reply:
fixed